manifests/t/TossPlace/TossPOS/0.0.6819/TossPlace.TossPOS.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: TossPlace.TossPOS
+PackageVersion: 0.0.6819
+PackageLocale: en-US
+Publisher: Toss Place Co., Ltd.
+PublisherUrl: https://tossplace.com/
+PublisherSupportUrl: https://tossplace.com/contact
+PrivacyUrl: https://pages.tossplace.com/policies/625/318
+PackageName: TossPOS
```

```suggestion
PackageName: 토스POS
```
This should match what is written to control panel